tracer/src/Datadog.Trace/Propagators/PropagationContextExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// <copyright file="PropagationContextExtensions.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Propagators;
+
+internal static class PropagationContextExtensions
+{
+    public static PropagationContext MergeBaggageInto(this PropagationContext context, Baggage destination)
```

Comment:
Nice 👍

Reply:
I usually don't like 1-line extension methods, but this one is a little different because it takes a `PropagationContext` and returns it back, while acting on the context's `Baggage`. This allows code like this which plays nice with what we already have everywhere:
```csharp
var context = SpanContextPropagator.Instance.Extract(headers).MergeBaggageInto(Baggage.Current);
```

(thanks @zacharycmontoya for the suggestion)